src/components/MovieCard/MovieCard.js

```diff
@@ -1,6 +1,43 @@
 import React from 'react';
 import './MovieCard.scss';
+import PropTypes from 'prop-types';
 
-export const MovieCard = () => (
-  <>Put the card here</>
+export const MovieCard = ({ imgUrl, title, description, imdbUrl }) => (
+  <>
+    <div className="card-image">
+      <figure className="image is-4by3">
+        <img
+          src={imgUrl}
+          alt="Film logo"
+        />
+      </figure>
+    </div>
+
+    <div className="card-content">
+      <div className="media">
+        <div className="media-left">
+          <figure className="image is-48x48">
+            <img src="./images/imdb-logo.jpeg" alt="imdb" />
+          </figure>
+        </div>
+
+        <div className="media-content">
+          <p className="title is-8">{title}</p>
+        </div>
+      </div>
+
+      <div className="content">
+        {description}
+        <br />
+        <a href={imdbUrl}>IMDB</a>
+      </div>
+    </div>
+  </>
 );
+
+MovieCard.propTypes = {
```

Comment:
You could use `import { TypeofValues } from '../../types';` here as well. I wrote smth like this and did work): 

`MovieCard.propTypes = TypeofValues.isRequired;`

Reply:
you-re right, it works))